src/IECoreHoudini/GEO_CortexPrimitive.cpp

```diff
@@ -455,18 +500,25 @@ void GEO_CortexPrimitive::copyPrimitive( const GEO_Primitive *src )
 	}
 	
 	const GEO_CortexPrimitive *orig = (const GEO_CortexPrimitive *)src;
-	const GA_IndexMap &srcPoints = orig->getParent()->getPointMap();
-	
+
 	/// \todo: should we make a shallow or a deep copy?
 	m_object = orig->m_object;
+
+	GEO_Primitive::copyPrimitive(src);
+}
+
+GEO_Primitive * GEO_CortexPrimitive::copy(int preserve_shared_pts) const
+{
+	GEO_CortexPrimitive *clone = static_cast<GEO_CortexPrimitive *>(GEO_Primitive::copy( preserve_shared_pts ));
+	if( !clone )
+	{
+		return nullptr;
+	}
+
+	/// \todo: should we make a shallow or a deep copy?
+	clone->m_object = m_object;
 	
-	GA_VertexWrangler vertexWrangler( *getParent(),	*orig->getParent() );
```

Comment:
Removing a virtual method would break compatibility, so we have to keep it unfortunately. Would be good to add a deprecation todo so we remember to remove it for Cortex 10